README.md

```diff
@@ -168,21 +235,34 @@ az keyvault set-policy --name '<YourKeyVaultName>' --object-id <PrincipalId> --s
 
 ## Deploy the Node App to Azure and retrieve the secret value
 
-Now that everything is set. Run the following command to deploy the app to Azure
+Now that everything is deployed, run the following command to deploy the app to Azure. This will push your local master branch to the git remote called 'azure' that you created earlier:
 
 ```
 git push azure master
 ```
 
-After this when you browse https://<app_name>.azurewebsites.net you can see the secret value.
-Make sure that you replaced the name <YourKeyVaultName> with your vault name
+When the git push command has completed you can now navigate to https://<AppName>.azurewebsites.net to see the secret value.
+
+Make sure that you replaced the name <AppName> with your vault name.
+
+# Folder Introduction
```

Comment:
Seems like this should be earlier in the doc.  Why moved to end?

Reply:
moved this earlier than "Prerequisites"